### PR TITLE
Release user delete api (v1.16.0)

### PR DESCRIPTION
**Pull request by wontheone1**
Release user delete api (v1.16.0)

### CODE REVIEW

**Comment by wontheone1** [comment]
https://github.com/sharetribe/flex-sdk-js/blob/master/docs/developing-sdk.md#release-a-new-version-to-npm

^ went through the above until step 3. I will do the rest steps (from 4 and on) after merging the PR.

**Review by bladealslayer — APPROVED** [review]
lgtm! I don't remember if these changes affect the build artifact, but you can try to run a build and see if that produces any diff. If it does, I'd either include the build in the branch or make it as separate commit directly to master as we've done in the past (see recent commit history).

**Pull request merged**
